web.browser/src/components/Posts/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import Post from './../Post';
+
+const Posts = ({ posts, updateVote, postFilter }) => {
+  const content = posts.filter((post) => (post.categories.includes(postFilter)))
+    .map(post => (
+      <Post
+        key={post.id}
+        author={post.author}
+        votes={post.votes}
+        categories={post.categories}
+        title={post.title}
+        description={post.description}
+        link={post.link}
+        updateVote={updateVote}
+        post={post}
```

Comment:
You'd almost be better off using non-jsx here, as every property of post maps to an attribute of the same name.

```js
React.createElement(Post, post);
```

You'd just have to make sure the object has a `key` property.